helm/values.yaml

```diff
@@ -36,6 +36,8 @@ providers:
     clientKey:
     tenantId:
     subscriptionId:
+    enableManagedIdentity: false
+    managedIdentityID:
```

Comment:
Could we merge these two into one `managedIdentityID`?

Reply:
sure, removed the bool flag